src/main/scala/ai/privado/utility/Utilities.scala

```diff
@@ -205,8 +205,9 @@ object Utilities {
     }
   }
 
-  def getTruncatedText(text: String, allowedCharLimit: Option[String]): String = {
-    allowedCharLimit.getOrElse("").toIntOption match {
+  def getTruncatedText(text: String, allowedCharLimit: Option[Int]): String = {
+    println("ppppppppp--- " + allowedCharLimit.getOrElse("nnnnn"))
```

Comment:
remove the print statement here. This would end up printing it several times on console

Reply:
yes already removed.